### ISSUE TITLE
[BUG] `required` keyword is not understood

### ISSUE THREAD

**Issue by luanpotter** [dart-code-checker/dart-code-metrics#214]
* **Dart code metrics version:** 2.4.0
* **Dart sdk version:** 2.12.0-259.16.beta

<details>
<summary>Configuration</summary>

```yaml
dart_code_metrics:
  rules:
    - prefer-trailing-comma
```

</details>

Before null-safety, we had the `@required` annotation; now that has been replaced by the `required` keyword.

However dart_code_metrics seem to not understand it; take this perfectly formatted trailing comma code:

```dart
int func({
  required int a,
  required int b,
}) {
  return a + b;
}
```

**What did you expect to happen?**

It should pass. in fact, if we replace the `required` keyword with the old annotation, it passes.

**What actually happened?**

We get an error!

```
lib/dcm_example.dart:
Warning Prefer trailing comma : 2:3 : prefer-trailing-comma https://git.io/Jkemi
```

**Are you willing to submit a pull request to fix this bug?**

Possibly. I never looked at the code but I can investigate (I appreciate help as well as maybe it might be an easy fix if you know your way around the code).

**Comment by incendial** [comment]
@luanpotter Hi, thank you for submitting the issue. Right now we are not supporting null-safety and, unfortunately, we blocked by analyzer_plugin package, that is used to connect to analysis server. But migration is on our roadmap.

**Comment by incendial** [comment]
Closing this due to a stable null safety release where this problem was resolved.
